test/test_app_logging.py

```diff
@@ -85,3 +93,40 @@ def fail():
                             azul_log.output[1],
                             'INFO:azul.chalice:Returning 500 response. To log headers and body, set AZUL_DEBUG to 1.'
                         )
+
+
+class TestPermittedWarnings(AzulUnitTestCase):
+
+    def test_permitted_warnings(self):
+        warnings.warn(" unclosed <ssl.SSLSocket fd=30, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM,"
+                      " proto=0, laddr=('192.168.1.11', 63179), raddr=('172.217.5.112', 443)>",
+                      category=ResourceWarning)
+        warnings.warn('Call to deprecated method fetch_bundle_manifest.)',
+                      category=DeprecationWarning)
+        warnings.warn("<class 'humancellatlas.data.metadata.api.LibraryPreparationProcess'> is deprecated",
+                      category=DeprecationWarning)
+
+
+class TestUnexpectedWarnings(TestCase):
+
+    def test_unexpected_warning(self):
+        msg = "Testing unexpected warnings, nothing to see here."
+
+        class TU(AzulUnitTestCase):
+
+            def test(self):
+                warnings.warn(message=msg,
+                              category=ResourceWarning)
+
+        test_case = TU('test')
+        test_suite = TestSuite()
+        test_result = TestResult()
+        test_suite.addTest(test_case)
+        test_suite.run(test_result)
+
+        self.assertEqual(test_result.testsRun, 1)
+        self.assertEqual(one(test_result.errors)[0].description,
```

Comment:
@hannes-ucsc during our discussion today we talked about usage of one() being applicable to the test_result (we only wanted 1 test to be run here), however we did not note that one() requires an `iterate` to be passed as an argument.

Reply:
What's an `iterate`? This looks ok to me.

Reply:
typo: `Iterable`